### ISSUE TITLE
(Mingw) Citra no longer compiles with SDL2 disabled

### ISSUE THREAD

**Issue by j-selby** [citra-emu/citra#3320]
Bisected to ab021d163e2a33b25b1811dc6f6abd924ec2d529 @lioncash 

Citra no longer compiles with ENABLE_SDL2 set to 0 on, at least, Mingw platforms.

CMake generation log: https://gist.github.com/j-selby/a61bfb10d544952a095d17cefcb2f50e
tl;dr:
`-DCMAKE_BUILD_TYPE=RelWithDebInfo -DENABLE_WEB_SERVICE=0 -DENABLE_QT=1 -DENABLE_SDL2=0 -G "CodeBlocks - MinGW Makefiles"`

CMake build log: https://gist.github.com/j-selby/3e9f9437e3c842bcb889b751a702f79f
tl;dr:
```
[ 31%] Building CXX object src/audio_core/CMakeFiles/audio_core.dir/sdl2_sink.cpp.obj
<path-to>\citra\src\audio_core\sdl2_sink.cpp:7:10: fatal error: SDL.h: No such file or directory
 #include <SDL.h>
          ^~~~~~~
compilation terminated.
```
Tested commit: `c143a5a3`
CMake version: `3.9.1`
Platform: `Windows 10.0.16299.125`

**Comment by yuriks** [comment]
Compile log with `make VERBOSE=1`?

**Comment by wwylele** [comment]
Same problem on Ubuntu, Cmake version 3.8.1
